crates/solidity/outputs/cargo/crate/src/generated/language.rs

```diff
@@ -4954,6 +6403,18 @@ impl Language {
         };
         let prefix_operator_parser = |input: &mut ParserContext| {
             ChoiceHelper::run(input, |mut choice, input| {
+                let result = parse_version_pragma_prefix_expression(input);
+                choice.consider(input, result)?;
+                let result = parse_version_pragma_prefix_expression(input);
+                choice.consider(input, result)?;
+                let result = parse_version_pragma_prefix_expression(input);
+                choice.consider(input, result)?;
+                let result = parse_version_pragma_prefix_expression(input);
+                choice.consider(input, result)?;
+                let result = parse_version_pragma_prefix_expression(input);
+                choice.consider(input, result)?;
+                let result = parse_version_pragma_prefix_expression(input);
+                choice.consider(input, result)?;
```

Comment:
It seems that precedence operator codegen is affected by this change. @OmarTawfik do you have an idea why that might be the case?

Reply:
Ah, I see we now have separate precedence operators per each terminal. This will benefit from the revamped parser generator interface since we're not limited by the v1 model anymore.